lib/src/widgets/chat_groupedlist_widget.dart

```diff
@@ -234,6 +256,39 @@ class _ChatGroupedListWidgetState extends State<ChatGroupedListWidget>
     );
   }
 
+  Future<void> _onReplyTap(String id, List<Message>? messages) async {
+    // Finds the replied message if exists
+    final repliedMessages = messages?.firstWhere((message) {
+      return id == message.id;
+    });
+
+    // Scrolls to replied message and highlights
+    if (repliedMessages != null && repliedMessages.key.currentState != null) {
+      await Scrollable.ensureVisible(
+        repliedMessages.key.currentState!.context,
+        // This value will make widget to be in center when auto scrolled.
+        alignment: 0.5,
+        curve:
+            widget.repliedMessageConfig?.highlightScrollCurve ?? Curves.easeIn,
+        duration: widget.repliedMessageConfig?.highlightDuration ??
+            const Duration(milliseconds: 300),
+      );
+      if (widget.repliedMessageConfig?.enableHighlightRepliedMsg == true) {
+        _replyId = id;
```

Comment:
Use widget.repliedMessageConfig?.enableHighlightRepliedMsg instead comparing bool

Reply:
this is null-able so we can't directly use `widget.repliedMessageConfig?.enableHighlightRepliedMsg` so rather then checking it's null-ability, I'm comparing it to `true` so its shorter.